tests/testthat/test-ancova.R

```diff
@@ -577,3 +400,245 @@ test_that("Field - Chapter 6 results match", {
   plot3 <-  results$state$figures[[3]]$obj$subplots$mainPlot
   jaspTools::expect_equal_plots(plot3, "PuppyLove3")
 })
+
+# Ordinal Restrictions ----
+# this test also tests anova restrictions
+options <- analysisOptions("Ancova")
+options$.meta <- list(contrasts = list(shouldEncode = TRUE), covariates = list(
+  shouldEncode = TRUE), customContrasts = list(shouldEncode = TRUE),
+  dependent = list(shouldEncode = TRUE), fixedFactors = list(
+    shouldEncode = TRUE), kruskalVariablesAssigned = list(
+      shouldEncode = TRUE), marginalMeansTerms = list(shouldEncode = TRUE),
+  modelTerms = list(shouldEncode = TRUE), moderatorFactorOne = list(
+    shouldEncode = TRUE), moderatorFactorTwo = list(shouldEncode = TRUE),
+  plotHorizontalAxis = list(shouldEncode = TRUE), plotSeparateLines = list(
+    shouldEncode = TRUE), plotSeparatePlots = list(shouldEncode = TRUE),
+  postHocTestsVariables = list(shouldEncode = TRUE), rainCloudPlotsHorizontalAxis = list(
+    shouldEncode = TRUE), rainCloudPlotsSeparatePlots = list(
+      shouldEncode = TRUE), restrictedModelMarginalMeansTerms = list(
+        shouldEncode = TRUE), simpleFactor = list(shouldEncode = TRUE),
+  wlsWeights = list(shouldEncode = TRUE))
```

Comment:
Thanks, I always forget to remove this. Perhaps we can drop this in jaspTools automatically, or is there ever a use-case when we need it?

Reply:
Well, jaspTools should also encode the column names. Perhaps we need it then? @JorisGoosen probably knows.

Reply:
To get `jaspTools` to behave the same as JASP it would need access to `meta` to know what to encode, so yes it would be needed then.

Reply:
Ok, so wouldn't it be prudent to just leave `meta` in the tests? Because when we get jaspTools to encode stuff at some point, would we need to go through all the test files to manually insert `meta` that we now manually delete?

Reply:
> Because when we get jaspTools to encode stuff at some point, would we need to go through all the test files to manually insert meta that we now manually delete?

I really hope not...

@JorisGoosen isn't `options$.meta` autogenerated/ can't we do also generate this automatically in R? The vast majority of unit tests don't have a .meta specified and it would be a hassle to add that.